projects/fsastorefrontlib/src/core/my-account/facade/quote.service.spec.ts

```diff
@@ -156,4 +209,29 @@ describe('QuoteServiceTest', () => {
       })
     );
   });
+
+  it('should be able to update quote', () => {
+    const priceAttributes = 'testPrice';
+    service.updateQuote(cartId, priceAttributes);
+    expect(store.dispatch).toHaveBeenCalledWith(
+      new fromAction.QuoteProcessAction({
+        userId: userId,
+        cartId: cartId,
+        action: QuoteActionType.UPDATE,
+        body: priceAttributes,
+      })
+    );
+  });
+
+  it('should be able to retrieve quotes', () => {
+    let user: string;
+    const quote = {
+      state: { code: 'BIND' },
+      cartCode: 'testCode',
+      userId: userId,
+    };
+    service.retrieveQuote(quote);
+    userIdService.getUserId().subscribe(data => (user = data));
```

Comment:
During subscription and listening emitted values you can expect something and than unsubscribe